Dockerfile

```diff
@@ -1,6 +1,14 @@
-FROM python:3.6-slim-stretch
-RUN pip install kubernetes==10.0.0 logstash_formatter>=0.5.17
+FROM ubuntu:focal
+RUN apt-get update -qq && \
+    apt-get install -y --no-install-recommends \
+        ca-certificates \
+        python3 \
+        python3-pip \
+    && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+RUN pip3 install --no-cache-dir kubernetes==12.0.1 logstash_formatter>=0.5.17
```

Comment:
I have some concern about the jump from 10.x to 12.x. I believe the vulnerability is also mitigated in a 10.x patch version which may be the safer change.

Reply:
Upgrading to `kubernetes==10.0.1` addresses CVE-2019-11324 (High), however CVE-2020-1747 (Critical) shows fixed version is 12.0.0a1.

When locally testing the upgrade to v10.0.1 I confirm that CVE-2020-1747 still shows up in the results:

```
$ trivy -q image --severity HIGH,CRITICAL kibana-sidecar

kibana-sidecar (ubuntu 20.04)
=============================
Total: 0 (HIGH: 0, CRITICAL: 0)


Python (python-pkg)
===================
Total: 1 (HIGH: 0, CRITICAL: 1)

+------------+------------------+----------+-------------------+---------------+--------------------------------------+
|  LIBRARY   | VULNERABILITY ID | SEVERITY | INSTALLED VERSION | FIXED VERSION |                TITLE                 |
+------------+------------------+----------+-------------------+---------------+--------------------------------------+
| kubernetes | CVE-2020-1747    | CRITICAL | 10.0.1            | 12.0.0a1      | PyYAML: arbitrary command            |
|            |                  |          |                   |               | execution through python/object/new  |
|            |                  |          |                   |               | when FullLoader is used              |
|            |                  |          |                   |               | -->avd.aquasec.com/nvd/cve-2020-1747 |
+------------+------------------+----------+-------------------+---------------+--------------------------------------+
```

v10.1.0 is also available, but that version introduces two more `PyYAML` Critical CVEs.

@ehearn should I proceed with downgrading to `kubernetes=10.0.1` for now and plan on addressing CVE-2020-1747 during another pass with enhanced testing?

Reply:
@nbarnumim thank you! I must have missed the CVE-2020-1747 item yesterday. In that case, I agree going to 12.x is the better choice. We will want to test it in one of the lower environments to make sure everything looks good.

@justinfiore could you possibly provide some guidance on the best way to confirm that the change of kubernetes version is working as expected?